x/fees/client/cli/query.go

```diff
@@ -0,0 +1,50 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/desmos-labs/desmos/x/fees/types"
+	"github.com/spf13/cobra"
+)
+
+// GetQueryCmd adds the query commands
+func GetQueryCmd(cdc *codec.Codec) *cobra.Command {
+	postQueryCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Querying commands for the fees module",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+	postQueryCmd.AddCommand(flags.GetCommands(
+		GetCmdQueryFeesParams(cdc),
+	)...)
+	return postQueryCmd
+}
+
+// GetCmdQueryFeesParams queries all the fees' module params
+func GetCmdQueryFeesParams(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "parameters",
```

Comment:
Probably it's better to name it as all the other modules
```suggestion
		Use:   "params",
```

Reply:
we use parameters in both profiles and posts modules, I think that in order to maintain the same name on every module we change them all or leave them like this

Reply:
Let's leave them params for now then